cmd/experimental/kjobctl/pkg/testing/wrappers/ray_cluster_spec_wrapper.go

```diff
@@ -0,0 +1,181 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package wrappers
+
+import (
+	rayv1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// RayClusterSpecWrapper wraps a RayClusterSpec.
+type RayClusterSpecWrapper struct{ rayv1.RayClusterSpec }
+
+// MakeRayClusterSpec creates a wrapper for a RayClusterSpec
+func MakeRayClusterSpec() *RayClusterSpecWrapper {
+	return &RayClusterSpecWrapper{
+		RayClusterSpec: rayv1.RayClusterSpec{},
+	}
```

Comment:
nit: Possible improvement. 2 lines less.

```suggestion
	return &RayClusterSpecWrapper{}
```

Reply:
Good point. Fixed.